Updated Intermediate HTML and CSS/Creating a Grid/Gap Section (#23786)
TheOdinProject/curriculum: html_css/grid-lessons/creating-a-grid.md

</p>
<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>

> Alternatively, we can set both the `row-gap` and `column-gap` by just using `gap`. For example: `gap: 10px 20px;` where the first value contains the `row-gap` and the second value contains the `column-gap`. Try using `gap` to change the `row-gap` and `column-gap` values with the example syntax. 

Reviewer: Wording suggestion:
```suggestion
Alternatively, we can set both the `row-gap` and `column-gap` by just using `gap`. For example: `gap: 10px 20px;` where the first value contains the `row-gap` and the second value contains the `column-gap`. Try using `gap` to change the `row-gap` and `column-gap` values in the above CodePen embed. 
```

> If we wanted the columns and rows to share the same gap value we can omit the `column-gap` property so it looks something like `gap: 25px;`. When a `column-gap` isn't specified, the value of the `column-gap` is assumed to be the same value as the `row-gap`.

Reviewer 2: ```suggestion
If we wanted the columns and rows to share the same gap value we can omit the `column-gap` property so it looks something like `gap: 25px;`. When a `column-gap` isn't specified, the value of the `column-gap` is assumed to be the same value as the `row-gap`.
```
